test/unit/view.js

```diff
@@ -289,6 +310,64 @@ describe('View class', () => {
       });
     });
 
+    describe('addClass()', () => {
+      let addClassToElementStub;
+      let addClassSpy;
+
+      beforeEach(() => {
+        addClassSpy = sinon.spy();
+        view.iframe = {addClass: addClassSpy};
+        addClassToElementStub = sinon.stub(elementClass, 'addClassToElement');
+      });
+
+      afterEach(() => {
+        addClassToElementStub.restore();
+      });
+
+      it('adds class to iframe if iframe exists', () => {
+        view.addClass('test-class');
+        assert.calledOnce(addClassSpy);
+        assert.calledWith(addClassSpy, 'test-class');
+        assert.notCalled(addClassToElementStub);
+      });
+
+      it('adds class to element if iframe does not exist', () => {
+        view.iframe = null;
+        view.addClass('test-class');
+        assert.calledOnce(addClassToElementStub);
+        assert.calledWith(addClassToElementStub, 'test-class');
+      });
+    });
+
+    describe('removeClass()', () => {
+      let removeClassFromElementStub;
+      let removeClassSpy;
+
+      beforeEach(() => {
+        removeClassSpy = sinon.spy();
+        view.iframe = {removeClass: removeClassSpy};
+        removeClassFromElementStub = sinon.stub(elementClass, 'removeClassFromElement');
+      });
+
+      afterEach(() => {
+        removeClassFromElementStub.restore();
+      });
+
+      it('removes class from iframe if iframe exists', () => {
+        view.removeClass('test-class');
+        assert.calledOnce(removeClassSpy);
+        assert.calledWith(removeClassSpy, 'test-class');
+        assert.notCalled(removeClassFromElementStub);
+      });
+
+      it('removes class from element if iframe does not exist', () => {
+        view.iframe = null;
+        view.removeClass('test-class');
+        assert.calledOnce(removeClassFromElementStub);
+        assert.calledWith(removeClassFromElementStub, 'test-class');
```

Comment:
Similar to the above, I think `removeClassFromElement` takes multiple parameters.